backend/curriculum_tracking/tests/test_activity_log_entry_creators_envoked.py

```diff
@@ -106,3 +106,45 @@ def test_add_review(self):
             start_url, data={"status": COMPETENT, "comments": "blah more stuff"}
         )
         self.assertEqual(LogEntry.objects.count(), 2)
+
+
+class log_card_review_requested_Tests(APITestCase, APITestCaseMixin):
+    LIST_URL_NAME = "agilecard-list"  #based on class of api-viewset
+    SUPPRESS_TEST_POST_TO_CREATE = True
+    SUPPRESS_TEST_GET_LIST = True
+
+    def test_review_requested(self):
```

Comment:
Your test names will need to be much more descriptive than this.  We not only want to know if the functionality of the api-view works we also need to know what are we supposed to get back.  For instance:

```
test_review_request_from_api_returns_card_in_review_status
```

I deliberately chose a long name to make the intentions of the test as clear as I can.  Although this name is perhaps not the best and can be improved upon, it does serve as an example of making the test name such so that the intentions of the test are evidently clear.

Reply:
I can see how it's good to have a very clear name of test intention. I had followed the convention of the first test written for another logging test class already there (i.e from the log_card_start_Tests class). So now I'm confused as to whether to follow the convention or amend the name of the test.....